### PR TITLE
test: extend retention policy query test

### PR DESCRIPTION
**Pull request by crepererum**
Add an ingester chunk to the parquet chunks.

Turns out we didn't apply retention policies to ingester chunks (this is only relevant for backfilling though).

Helpful for #6098.

### CODE REVIEW

**Comment by alamb** [comment]
CI still appears to be failing: https://app.circleci.com/pipelines/github/influxdata/influxdb_iox/31348/workflows/407c7f96-39c2-4b02-961d-9f602c086778/jobs/287794

**Comment by crepererum** [comment]
fixed

File: querier/src/ingester/mod.rs [hunk 1]
```diff
@@ -1208,6 +1226,8 @@ pub struct IngesterChunk {
 
     /// Summary Statistics
     summary: Arc<TableSummary>,
+
+    delete_predicates: Vec<Arc<DeletePredicate>>,
```
Comment (alamb) [inline querier/src/ingester/mod.rs:1230]: They are coming back!

**Review by alamb — APPROVED** [review]
makes sense to me

**Pull request merged**